web3.js/src/connection.ts

```diff
@@ -1826,6 +1826,11 @@ const GetLatestBlockhashRpcResult = jsonRpcResultAndContext(
   }),
 );
 
+/**
+ * Expected JSON RPC response for the "isBlockhashValid" message
+ */
+const IsBlockhashValidRpcResult = jsonRpcResult(boolean());
+
```

Comment:
I know that you're just following what others have done before, but can you _inline_ this into `isBlockhashValid`? It's only ever used once, not shared, and doing work (ie. calling `jsonRpcResult()` and `boolean()` in the module factory) is actually bad and going to be something we change in #20673.

Reply:
No problem, removed and inlined.